src/Akka.Persistence.PostgreSql.Tests/PostgreSqlSnapshotStoreSpec.cs

```diff
@@ -46,5 +50,20 @@ protected override void Dispose(bool disposing)
             base.Dispose(disposing);
             DbUtils.Clean();
         }
+
+        [Fact]
+        public void SnapshotStore_should_save_and_overwrite_snapshot_with_same_sequence_number()
```

Comment:
It should be, but this test is skipped in Sql.Common. I have written suggestions about Sql.Common here
https://github.com/akkadotnet/Akka.Persistence.SqlServer/pull/24